tests/config/state/azure/sql/statestore.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: statestore
+spec:
+  type: state.sqlserver
+  metadata:
+  - name: connectionString
+    value: ${{AzureSqlServerConnectionString}}
+  - name: tableName
+    value: dapr_conf_test
```

Comment:
Should the tableName also be dynamic to avoid run conflicts?

Reply:
It could be, but in general all the conformance tests are not parallelizable today with duplicate instances of themselves. 
The use of the env var just makes the framework look up the value in AKV for a value that is fixed at resource creation time and non-secret.